be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -182,7 +182,8 @@ Status KafkaDataConsumer::assign_topic_partitions(const std::map<int32_t, int64_
     RdKafka::ErrorCode err = _k_consumer->assign(topic_partitions);
     if (err) {
         LOG(WARNING) << "failed to assign topic partitions: " << ctx->brief(true) << ", err: " << RdKafka::err2str(err);
-        return Status::InternalError("failed to assign topic partitions");
+        std::string error_msg = "failed to assign topic partitions. " + _k_event_cb.get_error_msg();
```

Comment:
Can you put this error message template into error_code.h/cpp like ErrorCode.java?
It will be a good start for optimizing BE error messages.

What's the meaning of "assign topic partitions"?

Reply:
And, what is the `_k_event_cb`?